ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -195,6 +198,11 @@ open class PeoplePicker: BadgeField {
 
     open override func willMove(toWindow newWindow: UIWindow?) {
         super.willMove(toWindow: newWindow)
+        guard let newWindow else {
```

Comment:
Seems like the old logic in this function (on line 206) doesn't rely on the window not being nil. Are we sure we want to add the guard here or would it be safer to add it after the if block?

Reply:
Or maybe change the guard into an if let that updates the tokenSet if non nil?